index.html

```diff
@@ -129,8 +138,14 @@ <h3 class="lead text-center">
     </div>
 
     <div class="col-md-7">
-      <h2>JSX and React</h2>
-      <p>Babel has support for JSX and Flow. Check out our <a href="http://babeljs.io/docs/plugins/preset-react/">React preset</a> to get started. Use it together with the <a href="https://github.com/babel/babel-sublime">babel-sublime</a> package to bring syntax highlighting to a whole new level.</p>
+      <h2>JSX and Flow</h2>
+      <p>Babel can convert JSX syntax and strip out type annotations. Check out our <a href="http://babeljs.io/docs/plugins/preset-react/">React preset</a> to get started. Use it together with the <a href="https://github.com/babel/babel-sublime">babel-sublime</a> package to bring syntax highlighting to a whole new level.</p>
+{% highlight javascript %}
+// create a babel config with your presets
+echo '{ "presets": ["es2015"] }' > .babelrc
+{% endhighlight %}
+      <p>Learn about <a href="https://facebook.github.io/jsx/">JSX</a> and <a href="http://flowtype.org/">Flow</a>
```

Comment:
Even though the React docs are a mess, I'm thinking it'd be better to link JSX to https://facebook.github.io/react/docs/displaying-data.html#jsx-syntax or https://facebook.github.io/react/docs/jsx-in-depth.html, since the spec one here is more geared toward implementers. Unfortunate though that it does a better job of just showing an example right off the bat than the React docs.